lms/static/sass/views/_learner-profile.scss

```diff
@@ -23,6 +23,80 @@
         width: ($baseline*5);
     }
 
+    .profile-image-field {
+        @include float(left);
+
+        button {
+            background: transparent !important;
+            border: none !important;
+            padding: 0;
+        }
+
+        .u-field-image {
+            padding-top: 0;
+        }
+
+        .image-wrapper {
+            width: $profile-image-dimension;
+            position: relative;
+
+            .image-frame {
+                position: relative;
+                width: $profile-image-dimension;
+                height: $profile-image-dimension;
+            }
+
+            .u-field-upload-button {
+                width: $profile-image-dimension;
+                height: $profile-image-dimension;
+                position: absolute;
+                top: 0;
+                opacity: 0;
+
+                i {
+                    color: $white;
+                }
+            }
+
+            .upload-button-icon, .upload-button-title {
+                text-align: center;
+                transform: translateY(45px);
+                display: block;
+                color: $white;
+            }
+
+            .upload-button-input {
+                width: $profile-image-dimension;
+                height: 100%;
+                position: absolute;
+                top: 0;
+                left: 0;
+                opacity: 0;
+                cursor: pointer;
+            }
+
+            .u-field-remove-button {
+                width: $profile-image-dimension;
+                height: $baseline;
+                opacity: 0;
+                position: relative;
+                margin-top: 2px;
+                text-align: center;
+
+                &:focus, &:active {
+                    box-shadow: none;
+                    outline: 0;
+                }
+            }
+
+            &:hover {
+                .u-field-upload-button, .u-field-remove-button {
+                    opacity: 1;
```

Comment:
@cahrens and @andy-armstrong This should address the white text on light image issue. There should be a black transparent layer under the text on hover so any image will work.